code/game/objects/machinery/sentries.dm

```diff
@@ -1052,7 +1052,7 @@
 					if(S.opacity || S.density && S.throwpass == FALSE )
 						blocked = TRUE
 						break //LoF Broken; stop checking; we can't proceed further.
-			if(!blocked)
+			if(!blocked && !(M.status_flags & INCORPOREAL)) //Don't target incorporeals; we can't actually shoot them
 				targets += M
```

Comment:
why was this check moved

Reply:
So sentries can detect and report incorporeal Wraiths, but not try to shoot them which would be futile